docs/DeveloperGuide.md

```diff
@@ -322,15 +323,93 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case: Sort contacts**
+
+**Actor:** User
```

Comment:
Maybe we can standardise to not include? (Since base template states this is assumed)

Reply:
Not sure about this, honestly - my rationale was that it would be better to state it explicitly rather than leave it to assumptions, even if it seems obvious. What do the rest think?

Reply:
I think leaving this out is fine, since we only one type of actor. Unless the type of user need to be more specific, which we do not need for now.